index.d.ts

```diff
@@ -1921,6 +1982,21 @@ declare namespace Eris {
       /** @deprecated */
       DISCONNECT:          13;
     };
+    GuildScheduledEventStatus: {
+      SCHEDULED: 1;
+      ACTIVE:	2;
+      COMPLETED: 3;
+      CANCELED: 4;
+    };
+    GuildScheduledEventEntityTypes: {
+      STAGE_INSTANCE: 1;
+      VOICE: 2;
+      EXTERNAL: 3;
+    };
+    GuildScheduledEventPrivacyLevel: {
+      PUBLIC: 1;
```

Comment:
`PUBLIC` does not exist.

Reply:
Interesting, this was in the initial PR but removed in discord/discord-api-docs#4113. Have asked for clarification

Reply:
Alright so it seems like it was removed ~2 days after initial release. Will keep with convention and leave removed fields in constants